external-crates/move/crates/move-analyzer/editors/code/README.md

```diff
@@ -5,16 +5,44 @@ language [documentation](https://docs.sui.io/concepts/sui-move-concepts).
 
 # How to Install
 
-1. Open a new window in any Visual Studio Code application version 1.55.2 or greater.
+1. Open a new window in any Visual Studio Code application version 1.61.0 or greater.
 2. Open the command palette (`⇧⌘P` on macOS, or use the menu item *View > Command Palette...*) and
    type **Extensions: Install Extensions**. This will open a panel named *Extensions* in the
    sidebar of your Visual Studio Code window.
 3. In the search bar labeled *Search Extensions in Marketplace*, type **Move**. The Move extension 
    should appear as one of the option in the list below the search bar. Click **Install**.
 4. Open any file that ends in `.move`.
 
+Installation of the extension will also install a platform-specific pre-built move-analyzer binary in
+the `~/.sui/bin` directory, overwriting the existing binary if it already exists. The move-analyzer 
```

Comment:
References to `~` might throw off Windows users, the wording here probably needs a little finessing once Windows works, so feel free to address in follow up.

Reply:
Agreed! Since I managed to actually test this on Windows eventually, I also finessed this section to be more palatable for Windows users.